android/app/src/main/java/com/cascadebikeclub/MainApplication.java

```diff
@@ -4,6 +4,7 @@
 import android.content.Context;
 import com.facebook.react.PackageList;
 import com.facebook.react.ReactApplication;
+import com.oblador.vectoricons.VectorIconsPackage;
```

Comment:
I'm glad we are using VectorIconsPackage from AntDesign. I have used this icons before, in a different project and I liked them.

Reply:
It's a lot of icons including the AntDesign ones - https://oblador.github.io/react-native-vector-icons/